docs/using_kubernetes_events.md

```diff
@@ -1,28 +1,44 @@
 # Using Kubernetes Events
 
-Kubernetes events are objects that show you what is happening inside a cluster, such as what decisions were made by the controllers, in case an error occurs, or other messages. Most of the core components already create events through the API Server.
+Kubernetes events are objects that show you what is happening inside a cluster,
```

Comment:
It is kind of important to me to wrap the lines somehow instead of having super long lines nobody can really comment on.